sdk/build.gradle

```diff
@@ -10,14 +10,14 @@ ext {
 }
 
 android {
-    compileSdkVersion 29
-    buildToolsVersion "29.0.2"
+    compileSdkVersion "android-S" // TODO change this to 31 when Android 12 leaves beta
+    //buildToolsVersion "29.0.2"
     kotlinOptions {
         jvmTarget = "1.8"
     }
     defaultConfig {
         minSdkVersion 16
-        targetSdkVersion 29
+        targetSdkVersion 31
```

Comment:
We could potentially leave this as `targetSdkVersion: 29`. Though as a best practice, we may want to bump this to 31 since we support and test against Android 12.